recipe/meta.yaml

```diff
@@ -27,55 +30,60 @@ requirements:
     - future
     - menuinst  # [win]
   run:
-    - python
+    - python >=3.6
     - menuinst  # [win]
     - basemap >=1.2.0
     - chameleon
     - execnet
     - isodate
-    - jplephem
     - lxml
     - netcdf4
     - hdf4
     - pillow >=5,<6
-    - python-dateutil
     - pytz
-    - pyqt
-    - qt
+    - pyqt >=5,<6
+    - qt >=5,<6
     - requests
     - scipy
-    - sgp4
     - skyfield >=1.12
-    - six
     - tk
     - owslib
     - unicodecsv
     - fs_filepicker
-    - future
     - cftime >=1.0.1
     - matplotlib ==3.0.2
     - proj4 <6
     - flask
     - flask-httpauth
     - werkzeug
+    - flask-socketio
+    - flask-sqlalchemy
+    - passlib
+    - gitpython
+    - psycopg2
+    - validate_email
+    - multidict
```

Comment:
From https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=84745&view=logs:

```
export PREFIX=/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
export SRC_DIR=/home/conda/feedstock_root/build_artifacts/mss_1572258270941/test_tmp
import: 'mslib'
import: 'mslib'
+ mswms -h
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/bin/mswms", line 7, in <module>
    from mslib.mswms.mswms import main
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/mswms.py", line 33, in <module>
    from mslib.mswms.wms import mss_wms_settings
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/wms.py", line 57, in <module>
    from mslib.mswms.utils import conditional_decorator
  File "/home/conda/feedstock_root/build_artifacts/mss_1572258270941/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.6/site-packages/mslib/mswms/utils.py", line 30, in <module>
    import pint
ModuleNotFoundError: No module named 'pint'
```
I think you need:
```suggestion
    - multidict
    - pint
```

Reply:
thx, lets try. I missed that change